dev/publish_bom.py

```diff
@@ -115,10 +115,10 @@ def push_branch_and_tags(self):
     """
     major, minor, _ = self.__release_version.split('.')
 
-    # The stable branch will look like <major>.<minor>.X since nebula
+    # The stable branch will look like release-<major>.<minor>.x since nebula
     # enforces restrictions on what branches it does releases from.
     # https://github.com/nebula-plugins/nebula-release-plugin#extension-provided
-    stable_branch = '.'.join([major, minor, 'X'])
+    stable_branch = 'release-' + '.'.join([major, minor, 'x'])
```

Comment:
Since you use this new format twice it might save some grief in the future to put this logic in one place.